source/about/embed-mattermost-within-microsoft-teams.rst

```diff
@@ -1,20 +1,15 @@
-Install the Mattermost App in Microsoft Teams
-=============================================
-
+Embed Mattermost within Microsoft Teams
```

Comment:
Should this be "Embedded Mattermost in Microsoft Teams"? Deferring to you @cwarnermm

Reply:
"Embed" here is intended as a verb to align with the docs style guide. (Headings and titles aim to be imperative and start with a verb for consistency).